addons/pos_self_order/static/src/Pages/OrdersView/OrdersView.js

```diff
@@ -0,0 +1,18 @@
+/** @odoo-module */
+
+import { Component, onWillStart } from "@odoo/owl";
+import { useSelfOrder } from "@pos_self_order/SelfOrderService";
+import { Order } from "@pos_self_order/Components/Order/Order";
+import { NavBar } from "@pos_self_order/Components/NavBar/NavBar";
+import { Loading } from "@pos_self_order/Components/Loading/Loading";
+export class OrdersView extends Component {
+    static template = "pos_self_order.OrdersView";
+    static components = { Order, NavBar, Loading };
+    static props = [];
+    setup() {
+        this.selfOrder = useSelfOrder();
+        onWillStart(async () => {
+            this.selfOrder.updateOrders();
+        });
```

Comment:
What's the point of using `onWillStart` if you're not going to await the `updateOrders` call? Is this component dependent on the values that will be loaded by `updateOrders`?